packages/screens/Feed/FeedScreen.tsx

```diff
@@ -8,29 +8,62 @@ import { VideosFeedScreen } from "./VideosFeedScreen";
 import { FeedHeader } from "./components/FeedHeader";
 import { PostsRequest } from "../../api/feed/v1/feed";
 import { BrandText } from "../../components/BrandText";
-import { ScreenContainer } from "../../components/ScreenContainer";
+import {
+  ScreenContainer,
+  ScreenContainerProps,
+} from "../../components/ScreenContainer";
 import { MobileTitle } from "../../components/ScreenContainer/ScreenContainerMobile";
 import { NewsFeed } from "../../components/socialFeed/NewsFeed/NewsFeed";
 import { useForceNetworkSelection } from "../../hooks/useForceNetworkSelection";
 import { useIsMobile } from "../../hooks/useIsMobile";
 import { NetworkFeature } from "../../networks";
 import { ScreenFC } from "../../utils/navigation";
 
+export interface FeedTabScreenProps {
+  screenContainerOtherProps: Partial<ScreenContainerProps>;
+}
+
 export const FeedScreen: ScreenFC<"Feed"> = ({ route: { params } }) => {
   useForceNetworkSelection(params?.network);
   const isMobile = useIsMobile();
+  const screenContainerOtherProps: Partial<ScreenContainerProps> = {
+    fullWidth: true,
+    responsive: true,
+    noMargin: true,
+    noScroll: true,
+    footerChildren: <></>,
+    headerChildren: <BrandText>Social Feed</BrandText>,
+  };
 
   switch (params?.tab) {
     case "music":
-      return <MusicFeedScreen />;
+      return (
+        <MusicFeedScreen
+          screenContainerOtherProps={screenContainerOtherProps}
```

Comment:
If the props are all the same maybe you can create a base of `FeedScreen` that is juste a `ScreenContainer` with this props that just take a children ?
It's a proposal of course, and we can do it in an other PR 👍

Reply:
Ah yes of course